docs(README): add 'Export Multiple Conversations' section (#225)
pionxzh/chatgpt-exporter: README.md

```
</details>

## 📤 Export Multiple Conversations

When you click the "Export All" button, the **Export Conversations** dialog pops up. Here are the functions you can access.

**Export from official export file (conversations.json)**

Click the upload icon button to upload a JSON file of conversations, such as one downloaded from OpenAI.

**Export from API**

In the list of all your conversations, select which conversations you want to export. Check the "Select All" checkbox to export all your conversations.

> Choose the format you want to export to using the format pull-down selector under the list of your conversations at the left. You can choose from the following formats.

Reviewer: ```suggestion
Select your export format from the dropdown on the bottom left. You can choose from the following formats.
```
